src/vmm/src/version_map.rs

```diff
@@ -26,6 +28,7 @@ lazy_static! {
         let mut mapping = HashMap::new();
         mapping.insert(String::from("0.23.0"), 1);
         mapping.insert(String::from("0.24.0"), 2);
+        mapping.insert(String::from("0.25.0"), 3);
```

Comment:
Please also include the binary artifacts for running snapshot integration tests across versions.

Reply:
Once I get all approves, I will build and publish the artifacts so they are built from the same source as what is going to get merged.